test/rebar_paths_SUITE.erl

```diff
@@ -111,60 +121,95 @@ clashing_apps(Config) ->
 set_paths(Config) ->
     State = ?config(state, Config),
     RootDir = filename:split(?config(root_dir, Config)),
+    %% Take a snapshot of runtime deps being set; to see if your test is valid, this should fail
+    %% when you set the [deps] paths here
+    rebar_paths:set_paths([runtime], State),
+    RuntimePaths = code:get_path(),
+    %% Revert back to regular dep paths
+    rebar_paths:set_paths([deps, plugins], State),
```

Comment:
this line isn't necessary since the one after should replace deps,plugins with plugins,deps

Reply:
Gotcha